cmd/podman/main_remote.go

```diff
@@ -0,0 +1,43 @@
+// +build remoteclient
+
+package main
+
+import (
+	"os"
+
+	"github.com/containers/libpod/pkg/rootless"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const remote = true
+
+func init() {
+	//	remote client specific flags can go here.
+}
+
+func setSyslog() error {
```

Comment:
This is still sensible, even remote client only, I think

Reply:
I think it works exactly the same as it does local - just add the syslog endpoint to logrus

Reply:
per the meeting for the remote-client, it was removed.

Reply:
Alright, fine. I'm not firmly attached to it, but I think it's still useful even with remote.